spec.emu

```diff
@@ -73,20 +84,17 @@ contributors: Nicolò Ribaudo
         <emu-alg>
           1. If _P_ is a Symbol, then
             1. Return ! OrdinaryGet(_O_, _P_, _Receiver_).
-          1. <ins>Let _m_ be _O_.[[Module]].</ins>
-          1. <ins>If _m_ is not a Cyclic Module Record, then</ins>
-            1. <ins>Perform ? EvaluateSync(_m_).</ins>
-          1. <ins>Otherwise if _m_.[[Status]] is not ~evaluated~ and ReadyForSyncExecution(_m_) is *true*, then:</ins>
-            1. <ins>Perform ? EvaluateSync(_m_).</ins>
+          1. <ins>Perform ? EnsureDeferredNamespaceEvaluation(_O_).</ins>
           1. Let _exports_ be _O_.[[Exports]].
           1. If _exports_ does not contain _P_, return *undefined*.
-          1. <del>Let _m_ be _O_.[[Module]].</del>
+          1. Let _m_ be _O_.[[Module]].
           1. Let _binding_ be _m_.ResolveExport(_P_).
           1. Assert: _binding_ is a ResolvedBinding Record.
           1. Let _targetModule_ be _binding_.[[Module]].
           1. Assert: _targetModule_ is not *undefined*.
           1. If _binding_.[[BindingName]] is ~namespace~, then
-            1. Return GetModuleNamespace(_targetModule_).
+            1. Return GetModuleNamespace(_targetModule_, <ins>~evaluation~</ins>).
+            1. <ins>TODO: With `import defer * as x from "..."; export { x }`, the call above has to use ~defer~.</ins>
```

Comment:
Are you still planning to address this in this PR or as a follow-up?

Reply:
After further reading through the spec, I realized that this is already working.

When doing `import defer * as x from "..."; export { x }`, `_binding_.[[BindingName]]` will be already `x` and not `~namespace~` because it goes through the 10.a.ii.2 of [ParseModule](https://tc39.es/ecma262/#sec-parsemodule), and not 10.a.ii.3.